### PR TITLE
Add 30-Day Trial Content

### PR DESCRIPTION
**Pull request by sahilakos**
Update How it works, Pricing, FAQs, Checkout, TOS, and TOS Docx to reflect 30-day trial period for Gruntwork Subscription.

### CODE REVIEW

File: _data/faq.yml [hunk 1]
```diff
@@ -191,10 +191,12 @@
...
         <p>
-          So, you just want a sneak peek? Before any contracts are signed, we'd be happy to do a walkthrough of our
-          code, training, and process. You can also try out parts of the IaC Library for free by checking out our
-          <a href="/infrastructure-as-code-library/#open-source" title="Free Open Source modules by Gruntwork">open
-          source modules</a>.
+          Here are three ways you can try out Gruntwork Software with no commitment:
+          <ol>
+            <li>You can try out parts of the IaC Library for free by checking out our <a href="/infrastructure-as-code-library/#open-source" title="Free Open Source modules by Gruntwork">open source modules</a>.</li>
+            <li>Before any contracts are signed, we'd be happy to do a walkthrough of our code, training, and process.</li>
+            <li>We offer a 30-day money-back guarantee, so if for any reason you're not happy after signing up, we'll refund 100% of your money in the first 30 days. Check out "What if I want to cancel my Gruntwork Subscription?" for more info.</li>
```
Comment (brikis98) [inline _data/faq.yml]: Add link to the "What if I want to cancel" FAQ entry

File: _data/faq.yml [hunk 2]
```diff
@@ -229,24 +231,28 @@
...
-          access to all the code in the IaC Library, and we let you keep that code even if you cancel. If we didn't
-          have the annual contract requirement, customers could sign up for one day, get access to all the code, and
-          cancel immediately. The Dedicated Support Plan doesn't have these issues, so it's a monthly contract with
-          no annual commitment required.
+          The Gruntwork Subscription requires an annual contract (note: billing is monthly) because we give customers access to all the code in the IaC Library, and we let you keep that code even if you cancel. If we didn't have the annual contract requirement, customers could sign up for one day, get access to all the code, and cancel immediately. The Dedicated Support Plan doesn't have these issues, so it's a monthly contract with no annual commitment required.
+        </p>
+        <p>
+          However, we offer a 30-day money back guarantee, so if for any reason you're not happy after signing up, we'll let you out of the annual contract and refund 100% of your money in the first 30 days. Check out "What if I want to cancel my Gruntwork Subscription?" for more info.
```
Comment (brikis98) [inline _data/faq.yml]: Cross-link

File: _data/faq.yml [hunk 3]
```diff
@@ -229,24 +231,28 @@
...
+        </p>
+
+    - question: What if I want to cancel my Gruntwork Subscription?
+      answer: |
+        <p>
+          We offer a 30-day money-back guarantee. You can try out the Gruntwork Subscription, Dedicated Support, and the Reference Architecture, and if you're not happy for any reason, you can cancel within the first 30 days, and we'll let you out of the annual contract and refund 100% of your money.
+        </p>
+        <p>
+          Note that if you cancel in the first 30 days, you lose the license to the code in the Infrastructure as Code Library, so you must stop using it and delete any references to it; on the other hand, if you do stick with the annual subscription, and cancel after at least one year, you'll be allowed to keep all the code you had up to that point. See "What happens to my code if I cancel my subscription?" for more info.
```
Comment (brikis98) [inline _data/faq.yml]: cross-link

File: _data/faq.yml (hunk 1 above)
Comment (sahilakos) [reply _data/faq.yml]: Done

**Comment by brikis98** [comment]
Thank you! Will merge and deploy this now!

**Comment by sahilakos** [comment]
Added cross-links between all 3 FAQs: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/90/commits/6bc42741a7b825a782c826711458f112121ec2d2

**Pull request merged**